CONTRIBUTING.md

```diff
@@ -2,24 +2,28 @@
 
 ## Installation
 
-* `git clone <repository-url>`
-* `cd ember-aria`
-* `pnpm install`
+- `git clone <repository-url>`
+- `cd ember-aria`
+- `pnpm install`
 
 ## Linting
 
-* `pnpm lint`
-* `pnpm lint:fix`
+- `pnpm lint`
+- `pnpm lint:fix`
 
-## Running tests
+## To run docs and tests at the same time
 
-* `ember test` – Runs the test suite on the current Ember version
-* `ember test --server` – Runs the test suite in "watch mode"
-* `ember try:each` – Runs the test suite against multiple Ember versions
+```bash
+pnpm dev
+```
 
-## Running the test application
+## Running the tests
 
-* `ember serve`
-* Visit the test application at [http://localhost:4200](http://localhost:4200).
+- `pnpm dev` - an initial build is required to build the dist folder. You need to run this command at least once.
```

Comment:
Can we somehow automate the initial build? Like we do with `yarn-path` internally?